app/femr/common/models/PatientEncounterItem.java

```diff
@@ -37,9 +37,14 @@
 
     private String medicalDateOfVisit;
     private String physicianEmailAddress;
+    private String physicianFullName; //Andrew Change
 
     private String pharmacyDateOfVisit;
     private String pharmacistEmailAddress;
+    private String pharmacistFullName; //Andrew Change
+
+    private String nurseFullName; //Andrew Change
+
```

Comment:
We could then replace the below fields

private String physicianEmailAddress;
private String physicianFullName;

private String pharmacistEmailAddress;
private String pharmacistFullName;

private String nurseFullName;
private String nurseEmailAddress;

What do you think?